apps/website/src/pages/docs/alert.mdx

```diff
@@ -17,65 +17,78 @@ import * as AllExamples from 'examples';
 
-<LiveExample src='Alert.informational.tsx'>
-  <AllExamples.AlertInformationalExample client:load />
-</LiveExample>
-
-### Positive
-
-The positive alert is used to inform the user of a positive status.
-
-<LiveExample src='Alert.positive.tsx'>
-  <AllExamples.AlertPositiveExample client:load />
-</LiveExample>
-
-### Warning
-
-The warning alert is used to inform the user of something that is not currently disruptive to their work, but may be soon or eventually.
-
-<LiveExample src='Alert.warning.tsx'>
-  <AllExamples.AlertWarningExample client:load />
-</LiveExample>
+We provide:
 
-### Negative
+- Alert.Wrapper
+- Alert.Icon
+- Alert.Message
+- Alert.Action
+- Alert.CloseButton
 
-Use this only when the system is broken. Leave the technical details behind and explain the problem to the user in an easily understood tone. Mention when the problem might get solved, if it applies.
+The type of alert can be specified by utilizing the `type` prop inside `<Alert.Wrapper>`. There are four distinct types of alerts that can be chosen from:
 
-<LiveExample src='Alert.negative.tsx'>
-  <AllExamples.AlertNegativeExample client:load />
-</LiveExample>
-
-### Multilines
-
-All alerts may support multiple lines if necessary. We recommend keeping it brief and keep the message as concise as possible.
-
-## Placement
-
-The alert notification can display in two different ways:
+- positive : indication of a task or action that has been successfully completed. 
+- warning : requires attention but do not result in any immediate disruption.
+- negative : signify system outages or failures.
+- informational : intended to provide users with updates on non-disruptive tasks or work.
 
-### Inline
-
-An inline alert appears embedded within the copy of the site as a block element. This means that when the user scrolls, the alert scrolls as well.
-
-<LiveExample src='Alert.inline.tsx'>
-  <AllExamples.AlertInlineExample client:load />
+<LiveExample src='Alert.types.tsx'>
+  <AllExamples.AlertTypesExample client:load />
 </LiveExample>
 
-Inline alerts can optionally be dismissible. Once the alert is dismissed, the content below it will shift upwards due to the alert no longer consuming space.
-
-### Sticky
-
-A sticky alert appears full width stuck at the top of the window frame. That means if the user scrolls, the alert stays at the top of the frame and as the content scrolls underneath it. A sticky alert must always be dismissible. The alert appears above all other UI elements except for toast notifications and modals.
+#### `<Alert.Action>`
+
+By passing different props to `<Alert.Action>`, the user can interact with the alert to show additional information, redirected to other link or to close. A console message can be logged for event response.
+
+- When `as='a'` and `href` props are set, it behaves like a regular anchor link.
+- when `onClick` is passed, it behaves as a normal button.
+
+### Legacy API
```

Comment:
Not done. The "statuses" example should be moved out into a separate section, so that the "Composition" and "Legacy" examples are closer to each other. It is fine for `Alert.Action` to be below "Legacy".

Suggested order:
- h2: Usage
  - h3: Composition API
  - h3: Legacy API
  - h3: Alert.Action
- h2: Statuses
- h2: Sticky alert

Reply:
The text for Statuses section also needs to be moved